apps/geoportal/src/app/store/slices/measurements.ts

```diff
@@ -208,10 +224,12 @@ export const updateShapeById = (
 export const setLastVisibleShapeActive = () => {
   return function (dispatch, getState) {
     const state = getState();
-    const allShapes = state.measurements.shapes;
-    const lastShapeId = allShapes[allShapes.length - 1]?.shapeId;
-    if (lastShapeId) {
-      dispatch(setActiveShape(lastShapeId));
+    if (state.measurements) {
```

Comment:
optional remove

if, single store state should now always contain measurements obj